src/clr-icons/shapes/technology-shapes.ts

```diff
@@ -1112,7 +1118,10 @@ export const TechnologyShapes: any = {
     "hard-drive": ClrShapeHardDrive,
     "hard-drive-disks": ClrShapeHardDriveDisks,
     "nvme": ClrShapeNVMe,
-    "ssd": ClrShapeSSD
+    "ssd": ClrShapeSSD,
+    "bluetooth": ClrShapeBluetooth,
+    "no-bluetooth": ClrShapeNoBluetooth,
+    "process-on-vm": ClrShapeProcessOnVM
```

Comment:
I dont know what this icon represents. Is there a better name for this? @lil-kim

Reply:
That was for a VMW product. It was to be shown when there's a process running in a VM. We don't have to publish this to the external site.